plugins/flutter_aepcore/lib/src/aepmobile_init_options.dart

```diff
@@ -0,0 +1,29 @@
+class InitOptions {
+  String? appId;
+  String? filePath;
+  bool lifecycleAutomaticTracking;
+  Map<String, String>? lifecycleAdditionalContextData;
+  String? appGroupIOS;
+
+  // Constructor with named optional parameters
+  InitOptions({
+    this.appId,
+    this.filePath,
+    this.lifecycleAutomaticTracking = true, // Default value set to true
+    this.lifecycleAdditionalContextData = null, // Default value set to null
+    this.appGroupIOS = null, // Default value set to null
+  }) : assert(appId != null || filePath != null,
```

Comment:
If neither value is set, the SDK will default to the bundled configuration.
You should assert that both values are not set at the same time.